### PR TITLE
Allow the region to be passed in

### LINKED ISSUE #486: enable engineering in additional regions
suggest westeurope and australiasoutheast (recommendations based on availability of loganalytics service)

- [x] publish images in additional regions - try australia, germany, uk
- [x] parameterise eastus in create.sh, allow westeurope and australiasoutheast

at this point should be able to use published images outside of eastus

- [ ] copy published internal disk images to additional regions
- [ ] update purge and publish code accordingly

at this point should be able to use internal images outside eastus as well

### PR DESCRIPTION
**Pull request by asalkeld**
closes #486

### CODE REVIEW

File: hack/create.sh [hunk 1]
```diff
@@ -25,6 +25,23 @@ if [[ -z "$AZURE_CLIENT_SECRET" ]]; then
     exit 1
 fi
 
+if [[ -z "$AZURE_REGION" ]]; then
```
Comment (jim-minter) [inline hack/create.sh]: I know it's painful and unfriendly but I'd prefer to replace all this with

```
if [[ -z "$AZURE_REGION" ]]; then
    echo error: must set AZURE_REGION
    exit 1
fi
```

Fundamentally I want this script to die, not become a user friendly crutch.

Comment (jim-minter) [reply hack/create.sh]: Doing the above may imply that a separate commit is needed to fix up our tests.  If so, please have a go - look in openshift/release.

Comment (asalkeld) [reply hack/create.sh]: ok, I had this initially then thought of the build job... https://github.com/openshift/release/pull/1588

**Comment by asalkeld** [comment]
I must have missed something in release:

2018/09/20 23:26:46 Running pod e2e-azure
Using sync image registry.svc.ci.openshift.org/ci-op-7ks54fqk/stable:sync
+ set +x
error: must set AZURE_REGION
2018/09/20 23:27:01 Container setup in pod e2e-azure failed, exit code 1, reason Error

**Comment by kargakis** [comment]
/lgtm

Please send an email in aos-azure and ask folks to update their env files to use the new variable.

**Pull request merged**